frontend/src/components/Shared/CrossingMapPage/CrossingMapPage.js

```diff
@@ -149,12 +139,45 @@ class CrossingMapPage extends Component {
       selectedCommunity,
     } = this.state;
     const { currentUser } = this.props;
-    const allCommunities =
-      this.props.data &&
-      !this.props.data.loading &&
-      this.props.data.allCommunities
-        ? this.props.data.allCommunities.nodes
-        : null;
+
+    const isLoading =
+      !this.props.openCrossings ||
+      this.props.openCrossings.loading ||
+      !this.props.closedCrossings ||
+      this.props.closedCrossings.loading ||
+      !this.props.cautionCrossings ||
+      this.props.cautionCrossings.loading ||
+      !this.props.longtermCrossings ||
+      this.props.longtermCrossings.loading ||
+      !this.props.allCommunities ||
+      this.props.allCommunities.loading;
+
+    if (
+      !isLoading &&
+      (this.props.openCrossings.searchCrossings == null ||
+        this.props.closedCrossings.searchCrossings == null ||
+        this.props.cautionCrossings.searchCrossings == null ||
+        this.props.longtermCrossings.searchCrossings == null)
+    ) {
+      // TODO: add error logging
```

Comment:
Are we planning on using something like Sentry for error logging?

Reply:
I haven't looked into it too much yet, created an issue: https://github.com/cityofaustin/ctxfloods/issues/154